### PR TITLE
Added step to store resources in project/cluster to a clipboard

### PR DESCRIPTION
**Pull request by liangxia**

### CODE REVIEW

**Comment by yapei** [comment]
Thanks very much

**Comment by yapei** [comment]
@pruan-rht  Can you help review the changes? Thanks

File: features/step_definitions/resources.rb [hunk 1]
```diff
@@ -205,3 +205,15 @@
   cache_resources *list
   cache_resources cb[cb_name].first if cb[cb_name].first
 end
+
+Given /^(I|admin) stores? all (\w+)( in the#{OPT_QUOTED} project)? to the#{OPT_SYM} clipboard$/ do |who, type, in_project, namespace, cb_name|
+  cb_name ||= :resources
+  _user = who == "admin" ? admin : user
+
+  clazz = resource_class(type)
+  if in_project && BushSlicer::ProjectResource > clazz
```
Comment (pruan-rht) [inline features/step_definitions/resources.rb]: @liangxia what does this mean?  `BushSlicer::ProjectResource > clazz`

Comment (liangxia) [reply features/step_definitions/resources.rb]: @pruan-rht  In lib/world.rb, we have,
```
    RESOURCES.each do |clazz, snake_case|
      if clazz < ProjectResource
        eval <<-"END_EVAL", binding, __FILE__, __LINE__ + 1
          def #{snake_case}(*args, &block)
            project_resource(#{clazz}, *args, &block)
          end
        END_EVAL
      elsif clazz < ClusterResource
        eval <<-"END_EVAL", binding, __FILE__, __LINE__ + 1
          def #{snake_case}(*args, &block)
            cluster_resource(#{clazz}, *args, &block)
          end
        END_EVAL
      else
        raise "don't know how to create getter for #{clazz.name}"
      end
    end
```
I'm using it to determine whether it is a cluster resource or project resource.

**Pull request merged**